go/interpreter/lfvm/instructions.go

```diff
@@ -967,9 +905,13 @@ func opCreate2(c *context) {
 		value  = c.stack.pop()
 		offset = c.stack.pop()
 		size   = c.stack.pop()
-		salt   = c.stack.pop()
+		salt   = uint256.NewInt(0) // salt us zero for Create
```

Comment:
nit: declaring the salt as a 
```suggestion
		salt   tosca.Hash
```
and assigning the salt in line 911 below as
```
salt = c.stack.pop().Bytes32()
```
removes a memory allocation.